### DISCUSSION TITLE
Opening of the repo, today.

### DISCUSSION THREAD

**Discussion by fredcamaral** [LerianStudio/midaz#7] [Announcements]
We are excited to announce the opening of the repository for Midaz, an open-source ledger system designed to be multi-currency/asset, immutable, auditable, with a double-layer of security and ready to support integrations with multiple core banking and BaaS providers. Our team has deep experience in building core banking platforms in Latin America. In 2023, we founded Lerian to address what we called "inevitable churn" - the need for many players to internalize part of the infrastructure currently in the hands of SaaS providers.

Midaz aims to solve the challenges businesses face when managing financial transactions across various sectors by offering a flexible, open-source ledger system, enhanced security features, a modular architecture, smart contracts, plugins with different providers, native double-entry and reconciliation, and CLI support and a Console front-end.

We are roughly 80% ready to officially launch the product (expected July/August, 24). By opening up our codebase at this early stage, we invite developers, financial experts, and enthusiasts to contribute their ideas, expertise, and code to help shape the future of Midaz.

The whole concept of open-sourcing the Midaz codebase is to shift from charging for the Product to charging for the Service. We plan to monetize based on Enterprise Support, SaaS Hosting, and providing a Marketplace for different plugins/connectors.

Here's how you can get involved:

- Visit our [GitHub repository](github.com/LerianStudio/midaz) to access the Midaz codebase
- Join our [Discord community](https://discord.gg/Qp6ZNgYUsw) to discuss the project, share your ideas, and connect with other contributors
- Check out our [roadmap](https://docs.midaz.io/midaz/overview/about-midaz/roadmap-and-future-plans) to see the planned features and milestones for Midaz
- Contribute code, submit bug reports, or suggest improvements through our GitHub issues and pull requests
- Help spread the word about Midaz to your network and communities

We believe in the power of collaboration and community-driven development. We are excited to embark on this journey with you and create a ledger system that empowers businesses to unlock new opportunities and streamline their financial operations. Thank you for your support and interest in Midaz.

Best regards,
The Midaz Team at Lerian

**Comment by javanhaley** [comment]
I hope your launch is going well! I just came across some VERY HELPFUL insights to avoid "Inevitable Churn" 😁 in Dan Martell and the SaaS Academy team's new book Software As A Science. They  have tons of practical advice for startups and how to avoid many mistakes founders make in pricing, marketing etc. Not a long book and I loved it. Can get it for free right now at softwarebook.com.
